lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -317,6 +324,104 @@ func (c *ContainerProxy) CreateExecTask(handle string, config *types.ExecConfig)
 	return handleprime, eid, nil
 }
 
+func (c *ContainerProxy) InspectTask(op trace.Operation, handle string, eid string, cid string) (*models.TaskInspectResponse, error) {
```

Comment:
I don't see `cid` being used except for debug logging. If we're using op logging then this should be unnecessary as the calling function can log the cid.

See my other comments about dropping the parameterized error messages where it doesn't add value (it _does_ make parsing the errors in scripts harder).

Reply:
In this case it is also used in error message reporting. I can remove the logging, but we are still reporting this error message:
 `return nil, ConflictError(fmt.Sprintf("container (%s) has been poweredoff", cid))`

Which is then handed directly back to the docker client. I believe that I added the logging and I will remove it, but the `cid` is still needed to report the shutdown. 

I guess my other question to this is, do we need to include the container ID in the error message to begin with? The caller must know the target of `docker exec` to perform the operation. I am mainly thinking of automated scripts here that may have paramaterized inputs. Having the ID in the error message in this case would make it much easier for people to debug problems in their scripts.